core/src/main/java/com/scalar/db/sql/Example.java

```diff
@@ -0,0 +1,289 @@
+                  Assignment.column(COLUMN_NAME_3).value(Value.ofBigInt(100L)),
+                  Assignment.column(COLUMN_NAME_4).value(Value.ofFloat(1.23F)))
+              .build());
+
+      // Update
+      session.execute(
+          StatementBuilder.update(NAMESPACE_NAME, TABLE_NAME)
+              .set(
+                  Assignment.column(COLUMN_NAME_3).value(Value.ofBigInt(200L)),
+                  Assignment.column(COLUMN_NAME_4).value(Value.ofFloat(4.56F)))
+              .where(Condition.column(COLUMN_NAME_1).isEqualTo(Value.ofInt(10)))
+              .and(Condition.column(COLUMN_NAME_2).isEqualTo(Value.ofText("abc")))
+              .build());
+
+      // Select
+      ResultSet resultSet =
+          session.executeQuery(
+              StatementBuilder.select(COLUMN_NAME_1, COLUMN_NAME_2, COLUMN_NAME_3)
+                  .from(NAMESPACE_NAME, TABLE_NAME)
+                  .where(Condition.column(COLUMN_NAME_1).isEqualTo(Value.ofInt(10)))
+                  .and(Condition.column(COLUMN_NAME_2).isEqualTo(Value.ofText("abc")))
+                  .build());
+      for (Record record : resultSet) {
+        System.out.println("column1 value: " + record.getInt(COLUMN_NAME_1));
+        System.out.println("column2 value: " + record.getText(COLUMN_NAME_2));
+        System.out.println("column3 value: " + record.getBigInt(COLUMN_NAME_3));
+        System.out.println();
+      }
+
+      // Delete
+      session.execute(
+          StatementBuilder.deleteFrom(NAMESPACE_NAME, TABLE_NAME)
+              .where(Condition.column(COLUMN_NAME_1).isEqualTo(Value.ofInt(10)))
+              .and(Condition.column(COLUMN_NAME_2).isEqualTo(Value.ofText("abc")))
+              .build());
+
+      // Commit
+      session.commit();
+    } catch (SqlException e) {
+      session.rollback();
+    }
+
+    // Truncate a table
+    session.execute(StatementBuilder.truncateTable(NAMESPACE_NAME, TABLE_NAME).build());
+
+    // Truncate a coordinator table
+    session.execute(StatementBuilder.truncateCoordinatorTable().build());
+
+    // Drop an index
+    session.execute(
+        StatementBuilder.dropIndex()
+            .ifExists()
+            .onTable(NAMESPACE_NAME, TABLE_NAME)
+            .column(COLUMN_NAME_3)
+            .build());
+
+    // Drop a table
+    session.execute(StatementBuilder.dropTable(NAMESPACE_NAME, TABLE_NAME).ifExists().build());
+
+    // Drop a namespace
+    session.execute(StatementBuilder.dropNamespace(NAMESPACE_NAME).ifExists().cascade().build());
+
+    // Drop a coordinator table
+    session.execute(StatementBuilder.dropCoordinatorTable().ifExists().build());
+  }
+
+  public static void twoPhaseCommitTransactionModeExample(SessionFactory sessionFactory) {
+    System.out.println("Start two-phase commit transaction mode example");
+
+    Session session1 = sessionFactory.getTwoPhaseCommitTransactionSession();
+    Session session2 = sessionFactory.getTwoPhaseCommitTransactionSession();
```

Comment:
We can get a session for transaction mode from `SessionFactory.getTwoPhaseCommitTransactionSession()`. And we can also repeatedly reuse this session after committing or rolling back a transaction.